manifest.json

```diff
@@ -1,17 +1,17 @@
 {
-    "spark_version": "2.2.1",
+    "spark_version": "2.3.2",
     "default_spark_dist": {
         "hadoop_version": "2.7",
-        "uri": "https://downloads.mesosphere.com/spark/assets/spark-2.2.1-4-SNAPSHOT-bin-2.7.tgz"
+        "uri": "https://downloads.mesosphere.com/spark/assets/spark-2.3.2-1-SNAPSHOT-bin-2.7.3.tgz"
     },
     "spark_dist": [
         {
             "hadoop_version": "2.7",
-            "uri": "https://downloads.mesosphere.com/spark/assets/spark-2.2.1-4-SNAPSHOT-bin-2.7.tgz"
+            "uri": "https://downloads.mesosphere.com/spark/assets/spark-2.3.2-1-SNAPSHOT-bin-2.7.3.tgz"
```

Comment:
There's a separate ticket for porting CVEs commit from branch 2.2 to 2.3, in the current branch Hadoop is of version 2.7.3. It makes sense to (at least) update Hadoop dependency first in 2.3 branch and use the updated tar here to avoid having an outdated master in `spark-build`

Reply:
Okay let's port the CVE updates first and rebase this